### PR TITLE
chore: Migrate extensions into CCA (multiple-references-extension) [EXT-2566]

### PR DESCRIPTION
**Pull request by alexdonets**
#### Part 3/5 - `multiple-references-extension`

This PR is the 3rd of 5 PRs which implement the migration of the extensions from CCE into CCA.
In this PR we are migrating the `multiple-references-extension`.

#### Extensions migrated to CCA
- [x] `entry-extension-collapsible`
- [x] `markdown-extension`
- [ ] `multiple-references-extension`
- [ ] `rich-text-extension`
- [ ] `singleline-extension`

### CODE REVIEW

File: extensions/multiple-references-extension/package.json [hunk 1]
```diff
@@ -1,31 +1,50 @@
...
   "scripts": {
-    "start": "contentful-extension-scripts start",
-    "build": "contentful-extension-scripts build",
-    "deploy": "contentful-extension-scripts build && contentful extension update --force",
-    "configure": "contentful space use && contentful space environment use",
-    "login": "contentful login",
-    "logout": "contentful logout",
-    "help": "contentful-extension-scripts help"
+    "start": "SKIP_PREFLIGHT_CHECK=true cross-env BROWSER=none react-scripts start",
+    "build": "SKIP_PREFLIGHT_CHECK=true GENERATE_SOURCEMAP=false react-scripts build",
```
Comment (shikaan) [inline extensions/multiple-references-extension/package.json:16]: Same comment as the other PR

File: extensions/multiple-references-extension/src/index.tsx [hunk 2]
```diff
@@ -16,6 +17,7 @@ init<FieldExtensionSDK>((sdk) => {
         viewType="link"
         sdk={fieldSdk}
         isInitiallyDisabled={true}
+        hasCardEditActions={true}
```
Comment (shikaan) [inline extensions/multiple-references-extension/src/index.tsx:20]: Why was this added?

**Review by shikaan — COMMENTED** [review]
LGTM, same comment as the other PR + a question

File: extensions/multiple-references-extension/src/index.tsx (hunk 2 above)
Comment (alexdonets) [reply extensions/multiple-references-extension/src/index.tsx:20]: This was a mandatory prop in the component :/

File: extensions/multiple-references-extension/package.json [hunk 3]
```diff
@@ -1,31 +1,50 @@
...
   "scripts": {
-    "start": "contentful-extension-scripts start",
-    "build": "contentful-extension-scripts build",
-    "deploy": "contentful-extension-scripts build && contentful extension update --force",
-    "configure": "contentful space use && contentful space environment use",
-    "login": "contentful login",
-    "logout": "contentful logout",
-    "help": "contentful-extension-scripts help"
+    "start": "cross-env SKIP_PREFLIGHT_CHECK=true BROWSER=none react-scripts start",
+    "build": "SKIP_PREFLIGHT_CHECK=true GENERATE_SOURCEMAP=false react-scripts build",
```
Comment (shikaan) [inline extensions/multiple-references-extension/package.json:16]:
```suggestion
    "build": "cross-env SKIP_PREFLIGHT_CHECK=true GENERATE_SOURCEMAP=false react-scripts build",
```

**Review by shikaan — APPROVED** [review]

File: extensions/multiple-references-extension/package.json (hunk 3 above)
Comment (alexdonets) [reply extensions/multiple-references-extension/package.json:16]: Good catch! I will add it to all extensions/apps in my next PR!
